pkg/controller/operatingsystemconfig/actuator.go

```diff
@@ -117,6 +149,22 @@ func (a *actuator) handleReconcileOSC(_ *extensionsv1alpha1.OperatingSystemConfi
 		extensionsv1alpha1.Unit{Name: "locksmithd.service", Command: ptr.To(extensionsv1alpha1.CommandStop), Enable: ptr.To(false)},
 	)
 
+	if a.extensionConfig.UseNTP != nil && *a.extensionConfig.UseNTP {
```

Comment:
Should also implement the way back. That is disable ntpd and reenable systemd-timesyncd.